sycl/test-e2e/USM/dep_events.cpp

```diff
@@ -7,7 +7,7 @@
 //===----------------------------------------------------------------------===//
 // REQUIRES: aspect-usm_shared_allocations
```

Comment:
```suggestion
// REQUIRES: aspect-usm_shared_allocations
// UNSUPPORTED: cuda && windows
```
This may be a little clearer to the reader and would be skipped by LIT. Granted, it means we don't compile it, but we should be compiling it plenty for other targets.

Reply:
thanks @steffenlarsen. It can be compiled for windows / cuda, that's why I let it compiled. I can change it.

Reply:
I would prefer that. The intention of E2E tests should be to run the test, so having it "silently" skip running on CUDA Windows seems misleading.